coalib/settings/ConfigurationGathering.py

```diff
@@ -398,9 +405,31 @@ def gather_configuration(acquire_settings,
                                               acquire_settings,
                                               log_printer)
     save_sections(sections)
+    add_substring_targets(targets, sections)
```

Comment:
instead of adding artificial targets maybe we should change warn_nonexistent_targets to interpret the targets the right way?